packages/shared/src/components/modals/SubmitArticleModal.tsx

```diff
@@ -173,6 +174,17 @@ export default function SubmitArticleModal({
     );
   };
 
+  useEffect(() => {
+    trackEvent({
+      event_name: 'start submit article',
```

Comment:
Oh just one thing I noted now, we should move this to our enum names :)
It's a legacy one, but good opportunity to fix.

Possible also the feed_item_title.

Reply:
Good point. Let's update this.